tests/e2e/helpers.ts

```diff
@@ -45,7 +45,6 @@ async function archiveReports(buttons: Locator, page) {
         response.status() == 200
     );
 
-    await archiveButtons[0].click();
     await archivePromise;
     await archiveReports(buttons, page);
   }
```

Comment:
It looks like you lost the changes in this function when you resolved the merge conflict https://github.com/Enterprise-CMCS/macpro-mdct-mfp/pull/730/commits/0d79db3be162032c034b7383358495388a360d2c

Reply:
@gmrabian talked to Bangbay and I'm going to wait until her e2e work is merged in, use that +my steps for the new archiving process.